README.md

```diff
@@ -490,6 +491,24 @@ DESCRIPTION
 
 _See code: [src/commands/logout.js](https://github.com/apify/apify-cli/blob/v0.16.2/src/commands/logout.js)_
 
+## `apify pull [ACTORID]`
+
+Pulls the latest version of an actor from the Apify platform to the current directory.
+
+```
+USAGE
+  $ apify pull [ACTORID]
+
+ARGUMENTS
+  ACTORID  ID of an existing actor on the Apify platform which will be pulled. If not provided, the command will update
```

Comment:
@HonzaTuron We should mention you can use the Actor's name too. We do the same thing for the other commands

e.i.: `ACTID  Name or ID of the actor to run (e.g. "apify/hello-world" or "E2jjCZBezvAZnX8Rb")...`